### ISSUE TITLE
Building on Nvidia Jetson Nano

### ISSUE THREAD

**Issue by openoms** [openoms/raspiblitz#39]
https://www.nvidia.com/en-gb/autonomous-machines/embedded-systems/jetson-nano/

GPU | NVIDIA Maxwell™ architecture with 128 NVIDIA CUDA® cores
CPU | Quad-core ARM® Cortex®-A57 MPCore processor
Memory | 4 GB 64-bit LPDDR4
Storage | 16 GB eMMC 5.1 Flash
Video Encode | 4K @ 30 (H.264/H.265)
Video Decode | 4K @ 60 (H.264/H.265)
Camera | 12 lanes (3x4 or 4x2) MIPI CSI-2 DPHY 1.1 (1.5 Gbps)
Connectivity | Gigabit Ethernet
Display | HDMI 2.0 or DP1.2 \| eDP 1.4 \| DSI (1 x2) 2 simultaneous
UPHY | 1 x1/2/4 PCIE, 1x USB 3.0, 3x USB 2.0
I/O | 1x SDIO / 2x SPI / 6x I2C / 2x I2S / GPIOs
Size | 69.6 mm x 45 mm
Mechanical | 260-pin edge connector

Ubuntu 18.04 aarch64

**Comment by openoms** [comment]
Current most up-to-date branch to build form:
`wget https://raw.githubusercontent.com/openoms/raspiblitz/armbian1.2/build_sdcard.sh  && sudo bash build_sdcard.sh armbian1.2 openoms`

**Comment by openoms** [comment]
any progress on this @bavarianledger ?

**Comment by bavarianledger** [comment]
Yes, it's running smoothly on the Jetson Nano.

**Comment by openoms** [comment]
That is great to hear! 
Are you willing to share the link of the linux image you built on and maybe a part list, so we can add it to the tested hardware list?
Do you have a display on?
Did you use the latest state of the `armbian1.2` branch?

**Comment by bavarianledger** [comment]
I tried the Nvidia [Jetson Nano](https://www.nvidia.com/en-gb/autonomous-machines/embedded-systems/jetson-nano/) Dev Kit without any additional parts and no display connected to it. Starting with [Nvidia's Ubuntu 18.04 variant named L4T](https://developer.nvidia.com/embedded/dlc/jetson-nano-dev-kit-sd-card-image) 
I used the armbian1.2 branch right after my pull request.

**Comment by bavarianledger** [comment]
Are you still working on the `armbian1.2` branch?

**Comment by openoms** [comment]
> Are you still working on the `armbian1.2` branch?

Nope, it is the `x86v1.2` branch now. https://github.com/openoms/raspiblitz/tree/x86v1.2
I keep the latest working branch as default on https://github.com/openoms/raspiblitz.

**Comment by bavarianledger** [comment]
Thank you for clarification. The naming is a little bit confusing (aarch64/x86).
I'll give it a try on the Jetson Nano shortly.

**Comment by openoms** [comment]
Agreed, please forgive that. The aarch64 support is now merged in to the rootzoll's master. The X86 is the latest extra I am working on. The PR is in for this as well. There are lot of updates since the armbian1.2 like full update to the since published RaspiBlitz v1.2, Go specifically built for aarch63 as well etc.
Please let me know if you find anything broken.

**Comment by openoms** [comment]
You are probably familiar with the format now, but just to be sure here is the command to build the SD from this branch:

`wget https://raw.githubusercontent.com/openoms/raspiblitz/x86v1.2/build_sdcard.sh  && sudo bash build_sdcard.sh x86v1.2 openoms`

**Comment by openoms** [comment]
@bavarianledger do you have a case yet? https://geekworm.com/collections/new-arrivals/products/geekworm-nvidia-jetson-nano-metal-case-with-power-reset-control-switch

**Comment by bavarianledger** [comment]
> @bavarianledger do you have a case yet? https://geekworm.com/collections/new-arrivals/products/geekworm-nvidia-jetson-nano-metal-case-with-power-reset-control-switch

Wow, that's looking great. Ordering now. Thank you for the hint!
Do you have a Jetson Nano yet?

**Comment by openoms** [comment]
> Do you have a Jetson Nano yet?
It is hard to resist such great specs, but I think I have too many SBC-s already :grinning:

The Jetson Nano especially strong in the GPU, which I wouldn't use that much with the RaspiBlitz project.

**Comment by openoms** [comment]
Armbian is in the works for thw Jetson Nano 🎉 https://twitter.com/armbian/status/1143236524173135872?s=19
